tests/fuzz/oss_fuzz_build.sh

```diff
@@ -19,3 +19,8 @@ compile_go_fuzzer istio.io/istio/tests/fuzz FuzzParseAndBuildSchema fuzz_parse_a
 compile_go_fuzzer istio.io/istio/tests/fuzz FuzzConfigValidation fuzz_config_validation
 compile_go_fuzzer istio.io/istio/tests/fuzz FuzzParseMeshNetworks fuzz_parse_mesh_networks
 compile_go_fuzzer istio.io/istio/tests/fuzz FuzzValidateMeshConfig fuzz_validate_mesh_config
+compile_go_fuzzer istio.io/istio/tests/fuzz FuzzAnalyzer fuzz_analyzer
+
+
+# Create seed corpora:
+zip "${OUT}"/fuzz_analyzer_seed_corpus.zip "${SRC}"/istio/galley/pkg/config/analysis/analyzers/testdata/*.yaml
```

Comment:
nice! should it be `${SRC}"/galley/...` though? I would have thought SRC=gopath/src/istio.io/istio, but maybe not?

Reply:
istio's path in the OSS-fuzz environment is $SRC/istio: https://github.com/google/oss-fuzz/blob/master/projects/istio/Dockerfile#L20